client/web/compose/src/components/PageBlocks/ChartBase.vue

```diff
@@ -42,18 +42,32 @@
         </b-form-group>
 
         <template #modal-footer="{ ok }">
-          <b-button
-            variant="light"
-            @click="resetLiveFilter()"
+          <div
+            class="d-flex justify-content-between align-items-center w-100"
           >
-            Reset
-          </b-button>
-          <b-button
-            variant="primary"
-            @click="ok()"
-          >
-            Save and Close
-          </b-button>
+            <b-button
+              variant="light"
+              @click="resetLiveFilter()"
```

Comment:
Reset would should reset to original value that is saved in the chart configurator.